### PR TITLE
Add io.github.cxong.cdogs-sdl

### PR DESCRIPTION
**Pull request by cxong**

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build io.github.cxong.cdogs-sdl

**Comment by cxong** [comment]
```
Error: icon cdogs-sdl not found below /srv/buildbot/worker/build-x86_64-4/build/.flatpak-builder/rofiles/rofiles-a7hxyL/files/share/icons
```
Looks like the icon is missing; where do I need to provide them? Usually they are in e.g. `share\icons\hicolor\128x128\apps\io.github.cxong.cdogs-sdl.png`

File: io.github.cxong.cdogs-sdl.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop">
```
Comment (bilelmoussaoui) [inline io.github.cxong.cdogs-sdl.appdata.xml]:
```suggestion
<component type="desktop-application">
```

File: io.github.cxong.cdogs-sdl.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  <id>cdogs-sdl.desktop</id>
```
Comment (bilelmoussaoui) [inline io.github.cxong.cdogs-sdl.appdata.xml]:
```suggestion
  <id> io.github.cxong.cdogs-sdl</id>
```

Comment (bilelmoussaoui) [reply io.github.cxong.cdogs-sdl.appdata.xml]: You need a `launchable` tag here too.

File: io.github.cxong.cdogs-sdl.desktop [hunk 3]
```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Name=C-Dogs SDL
+GenericName=Action/Arcade Game
+Version=0.7.2
```
Comment (bilelmoussaoui) [inline io.github.cxong.cdogs-sdl.desktop]:
```suggestion
Version=0.7.2
```
The version here is for the Spec version and not the app version.

File: io.github.cxong.cdogs-sdl.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "cdogs-sdl",
+  "rename-icon": "cdogs-sdl",
+  "finish-args": [
+      "--device=dri",
+      "--share=ipc",
+      "--socket=pulseaudio",
+      "--socket=wayland",
```
Comment (bilelmoussaoui) [inline io.github.cxong.cdogs-sdl.json]: Does it work on wayland?

**Comment by bilelmoussaoui** [comment]
The desktop file and the metainfo file should go upstream.

**Comment by LeandroStanger** [comment]
bot, build io.github.cxong.cdogs-sdl

**Comment by cxong** [comment]
Thanks; will add fixes to upstream and submit again later

**Pull request closed without merging**